roles/dnsmasq/tasks/main.yml

```diff
@@ -108,6 +108,17 @@
     - timeout:2
     - attempts:2
 
+- name: Remove search and nameserver options from resolvconf base
+  lineinfile:
+    dest: /etc/resolvconf/resolv.conf.d/base
+    state: absent
+    regexp: "^{{ item }}.*$"
+    backup: yes
+    follow: yes
+  with_items:
+    - search
+    - nameserver
```

Comment:
It doesn't. I tested by adding a search and nameserver line to my "base" resolvconf file, ran resolvconf -u to show they get added, then applied dnsmasq role, and confirmed they are all gone.